spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -211,4 +211,10 @@ public boolean handleTimestampWithoutZone() {
         .defaultValue(SparkSQLProperties.HANDLE_TIMESTAMP_WITHOUT_TIMEZONE_DEFAULT)
         .parse();
   }
+
+  public Long fromTimestamp() {
```

Comment:
I think this needs a better name. It isn't clear what `options.fromTimestamp()` means in other contexts.

Reply:
How about `streamFromTimestamp` ?